src/ice_shelf/MOM_ice_shelf_initialize.F90

```diff
@@ -125,22 +125,22 @@ subroutine initialize_ice_thickness_from_file(h_shelf, area_shelf_h, hmask, G, U
   call get_param(PF, mdl, "ICE_AREA_VARNAME", area_varname, &
                  "The name of the area variable in ICE_THICKNESS_FILE.", &
                  default="area_shelf_h")
-
+  hmask_varname="h_mask"
   if (.not.file_exists(filename, G%Domain)) call MOM_error(FATAL, &
        " initialize_topography_from_file: Unable to open "//trim(filename))
   call MOM_read_data(filename, trim(thickness_varname), h_shelf, G%Domain, scale=US%m_to_Z)
   call MOM_read_data(filename,trim(area_varname), area_shelf_h, G%Domain, scale=US%m_to_L**2)
-
+  call MOM_read_data(filename, trim(hmask_varname), hmask, G%Domain)
   isc = G%isc ; jsc = G%jsc ; iec = G%iec ; jec = G%jec
 
+  if (len_sidestress > 0.) then
```

Comment:
Please indent the block of code inside of this if-test by 2 points, consistent with MOM6 code style standards.  This may also be a case where the `do j` and `do i`  loops should be combined onto a single line with a semi-colon, as they share the same body.

Reply:
This is  a step in the right direction, but the code in the if-block on lines 155-164 also needs to be indented.

Reply:
Thank you for catching this. Corrected.